linux/LinuxProcessList.c

```diff
@@ -1821,6 +1824,10 @@ static inline void LinuxProcessList_scanMemoryInfo(ProcessList* this) {
             break;
          }
          break;
+      case 'Z':
+	      tryRead("Zswap:", zswapCompMem);
+	      tryRead("Zswapped:", zswapOrigMem);
+	      break;
```

Comment:
Indentation

Reply:
Fixed, along with the rest of the indentation fails. Sorry for the noise, my editor didn't apply the editorconfig file entirely correctly and I just missed all the tabs.